wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -336,6 +337,15 @@ public final class ZMUserSession: NSObject {
             context: syncContext)
     }()
 
+    public lazy var selfMLSClientID: MLSClientID? = {
+        guard let selfUserClient  else {
+            return nil
+        }
+       return  MLSClientID(userClient: selfUserClient)
+    }()
+
+    public var lastE2EIUpdateDate: LastE2EIdentityUpdateDateProtocol?
```

Comment:
This shouldn't be modifiable from outside.
```suggestion
    public private(set) var lastE2EIUpdateDate: LastE2EIdentityUpdateDateProtocol?
```